pandas/tests/tslibs/test_array_to_datetime.py

```diff
@@ -132,7 +132,7 @@ def test_coerce_outside_ns_bounds(invalid_date, errors):
     if errors == "raise":
         msg = "^Out of bounds nanosecond timestamp: .*, at position 0$"
 
-        with pytest.raises(ValueError, match=msg):
+        with pytest.raises(OverflowError, match=msg):
```

Comment:
this should probably be OutOfBoundsDatetime (which would pass ATM bc it subclasses ValueError)